content/it/docs/concepts/architecture/control-plane-node-communication.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Comunicazione Piano di controllo - Nodo
+content_type: concept
+weight: 20
+aliases:
+- master-node-communication
+---
+
+<!-- overview -->
+
+Questo documento cataloga i percorsi di comunicazione tra il piano di controllo (_control plane_), in realtà l'apiserver, e il cluster Kubernetes. L'intento è di consentire agli utenti di personalizzare la loro installazione per rafforzare la configurazione di rete affinché il cluster possa essere eseguito su una rete non affidabile (o su IP completamente pubblici resi disponibili da un fornitore di servizi cloud).
```

Comment:
```suggestion
Questo documento cataloga le connessioni tra il piano di controllo (_control plane_), in realtà l'apiserver, e il cluster Kubernetes. L'intento è di consentire agli utenti di personalizzare la loro installazione affinché il cluster possa essere eseguito su una rete publica (o su IP resi disponibili da un fornitore di servizi cloud).
```
Proposing some changes to make it more fluent...

Reply:
fixed